DigitalLearningSolutions.Data.Tests/DataServices/UserDataServiceTests/AdminUserDataServiceTests.cs

```diff
@@ -12,16 +12,43 @@
     public partial class UserDataServiceTests
     {
         [Test]
-        public void GetAdminById_Returns_admin_user()
+        public void GetAdminById_returns_admin()
         {
             // Given
             var expectedAdminEntity = UserTestHelper.GetDefaultAdminEntity();
 
             // When
-            var returnedAdminUser = userDataService.GetAdminById(7);
+            var returnedAdminEntity = userDataService.GetAdminById(7);
 
             // Then
-            returnedAdminUser.Should().BeEquivalentTo(expectedAdminEntity);
+            returnedAdminEntity.Should().BeEquivalentTo(expectedAdminEntity);
+        }
+
+        [Test]
+        public void GetAdminById_returns_admin_with_correct_user_centre_details()
+        {
+            using var transaction = new TransactionScope();
+
+            // Given
+            connection.Execute(
+                @"INSERT INTO UserCentreDetails (UserID, CentreID, Email)
+                    VALUES (2, 2, 'centre@email.com')"
+            );
+            var expectedUserCentreDetails = UserTestHelper.GetDefaultAdminEntity(
+                userCentreDetailsId: 1,

```

Comment:
I think it is wise to not assert that the ID must match some specific value. As we are setting the Id here in the expectedUserCentreDetails, would it be helpful to add a comment in the assertions to state why we aren't comparing it, and why we are setting it here? Or do we think it is obvious enough as is?

Reply:
Probably needs a comment at it isn't obvious behaviour.